tests/integration/tracing_test.go

```diff
@@ -70,7 +70,9 @@ func TestTracing(t *testing.T) {
 
 	select {
 	case <-etcdSrv.Server.ReadyNotify():
-	case <-time.After(1 * time.Second):
+	case <-time.After(5 * time.Second):
+		// default randomized election timeout is 1 to 2s, single node will fast-forward 900ms
+		// change the timeout from 1 to 5 seconds to ensure de-flaking this test
```

Comment:
The change makes sense to me. But could you please clarify what does "single node will fast-forward 900ms" mean?

Reply:
```
    logger.go:130: 2023-03-16T02:01:04.478Z	INFO	default	started as single-node; fast-forwarding election ticks	{"local-member-id": "8e9e05c52164694d", "forward-ticks": 9, "forward-duration": "900ms", "election-ticks": 10, "election-timeout": "1s"}
```

Theoretically maximum wait time should be `2s - 900ms`

Reply:
thx for the clarification.